### ISSUE TITLE
no button to add new users

### ISSUE THREAD

**Issue by bdparrish** [bdparrish/cmc2#12]

**Comment by parrishd** [comment]
yeah. good point. I designed this originally thinking that the users would be forced to register, but now we are storing these unregistered monitors in the same table to track monitoring hours. I'll work on it.
